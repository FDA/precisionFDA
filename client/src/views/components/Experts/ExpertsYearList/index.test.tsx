import React from 'react'
import { shallow } from 'enzyme'

import { ExpertsYearList } from '.'


describe('ExpertsYearList', () => {
  it('matches snapshot', () => {
    const wrapper = shallow(<ExpertsYearList setYearHandler={() => {}} />)

    expect(wrapper).toMatchSnapshot()
  })
})