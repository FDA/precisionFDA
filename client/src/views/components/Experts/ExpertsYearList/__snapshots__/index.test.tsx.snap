// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ExpertsYearList matches snapshot 1`] = `
<QueryList
  className="year-list"
  emptyMessage="No previous experts"
  listExtractor={[Function]}
  query={[Function]}
  template={[Function]}
/>
`;
